style-dictionary/visual-refresh/colors.ts

```diff
@@ -266,16 +266,6 @@ const tokens: StyleDictionary.ColorsDictionary = {
   colorDropzoneBackgroundHover: { light: '{colorBlue200}', dark: '{colorBlue600}' },
   colorDropzoneTextActive: { light: '{colorGrey550}', dark: '{colorGrey800}' },
   colorDropzoneTextHover: { light: '{colorBlue800}', dark: '{colorWhite}' },
-  colorBackgroundSeverityCritical: '{colorChartsStatusCritical}',
-  colorTextSeverityCritical: { light: '{colorGrey100}', dark: '{colorBlack}' },
-  colorBackgroundSeverityHigh: '{colorChartsStatusHigh}',
-  colorTextSeverityHigh: { light: '{colorGrey100}', dark: '{colorGrey900}' },
-  colorBackgroundSeverityMedium: '{colorChartsStatusMedium}',
-  colorTextSeverityMedium: '{colorGrey900}',
-  colorBackgroundSeverityLow: '{colorChartsStatusLow}',
```

Comment:
Since these tokens were assigned to aliases that are themeable (colorChartsStatusX), and the new tokens use different value assignments, this means anyone who previously themed the chart severity colors will get overridden by this change, and need to reassign those in order to see them reflected in the badge component. This could be seen as a feature or a bug/visually breaking change, depending on how you look at it. 

Why are we not updating the severity colors at the source (colorChartStatusX)?

Reply:
We can't use the `colorChartStatusX` here, since those colors were not meant to have text on top of them. So this purposefully separates the chart severity colors (standalone) with notification severity colors (used with text).

Interesting point about the potential breaking change, though. Any recommendations on how to proceed here?

Reply:
Investigated offline. The reason chart colors need to remain separate is for a couple reasons: 
1. chart segments are considered graphical objects where the surface area of the background or boundaries of the segments communicate essential information, and
2. while we recommend it, we cannot guarantee that a visible label _and_ value will always be present on charts at all times, especially for line/bar charts.

This means, the chart colors need to meet 3:1 against the background color of containers, according to [WCAG SC 1.4.11](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html). And the new colors being introduced here for low and medium severity (yellow and orange) would not meet the 3:1 contrast ratio with pure white. 
Badges are exempt from this because the boundaries of the badge is not conveying essential information, and thus does not need to meet 3:1 with the background. The only ratio requirements that needs to be met are the badge text with the badge background, which passes for all new colors.